src/NServiceBus.AcceptanceTests/Basic/When_deferring_to_non_local.cs

```diff
@@ -13,7 +13,13 @@ public void Message_should_be_received()
             var context = new Context();
 
             Scenario.Define(context)
-                    .WithEndpoint<Endpoint>(b => b.Given((bus, c) => bus.Send(new MyMessage(), new SendOptions(delayDeliveryFor: TimeSpan.FromSeconds(3)))))
+                    .WithEndpoint<Endpoint>(b => b.Given((bus, c) =>
+                    {
+                        var options = new SendOptions();
+
+                        options.DelayDeliveryWith(TimeSpan.FromSeconds(3));
```

Comment:
Just a random idea: Would there be a possibility to simulate time going by to not wait for the deferral? Probably it is a non issue because we have not many of those tests